### ISSUE TITLE
unmet peer vite@^3.2.3: found 4.0.0

### ISSUE THREAD

**Issue by n8sabes** [wmertens/styled-vanilla-extract#4]
WARN  Issues with peer dependencies found
.
└─┬ styled-vanilla-extract 0.5.0
  └── ✕ unmet peer vite@^3.2.3: found 4.0.0

https://github.com/wmertens/styled-vanilla-extract/blob/674770977af4d25c9ba656ff5700b9d0a4693752/package.json#L78

**Comment by wmertens** [comment]
This is now working, right?

**Comment by n8sabes** [comment]
I just did a project update `pnpm up --latest` and get the warning below. My package is using `"vite": "4.0.3"`.

```
├─┬ @vanilla-extract/vite-plugin 3.7.0
│ └── ✕ unmet peer vite@"^2.2.3 || ^3.0.0": found 4.0.3
└─┬ styled-vanilla-extract 0.5.3
  └── ✕ unmet peer vite@"^3.2.3 | ^4": found 4.0.3
```

**Comment by wmertens** [comment]
Thanks for your fix in #6
